src/react-app-env.d.ts

```diff
@@ -10,7 +10,5 @@ declare module 'ndarray-unpack' {
 }
 
 declare module 'ndarray-ops' {
-  function assign(a: ndarray<T>, b: ndarray<T>);
-
-  export = { assign };
+  export function assign(a: ndarray<T>, b: ndarray<T>);
```

Comment:
It does not change the export from `{ assign }` to `assign` ?

Reply:
Before you were declaring a default export equal to an object with an `assign` property -- now, `assign` is a named export. This allows the  use of `import { assign }` instead of `import ops` and then `ops.assign`.

Reply:
All right, understood